docs/070-project-management/pm-unanet-tasks.md

```diff
@@ -0,0 +1,255 @@
+   - You can also enter the labor category here if it is the same for each person
+
+## Adjust projections for holidays or other time-off
+
+In general, resource allocations can be expected using the following rations:
+
+1. If a FT resource devoted to the project, then 80%
+2. If an AD resource devoted to the project, then 65%
+3. If a PT resource devoted to the project, then 40%
+
+From that place, then deduct - monthly - for any of the following:
+
+1. holidays
+2. OOO
+3. Any other events, e.g., summits, etc.
+
+Pro Tip: For finding OOO dates on the calendar, use the CivicActions shared calendar and try searching "name OOO". This will allow you to isolate the forthcoming dates for the given person.
+
+## Reviewing and approving expenses
+
+1. Find your approvals
+
+   - Click on the Unanet logo
+   - Scroll down to Approvals
+   - Look for Expense Reports and/or Requests and click on Primary Approvals to view
+
+![Screenshot of Approving Expense 1](../images/Approving-Expenses-1.png)
+
+NOTE: All expense reimbursements except for ProDev < $50 should be submitted as a REQUEST first.
+
+To check this quickly:
+
+      i. Click on the magnifying glass next to the expense report
+      ii. Scroll down to Approval History
+      iii. A report that has previously been a request will show 'PreApproving' and 'PreApproved' statuses in the history. If it has never been a request, you will only see 'Approving' and/or 'Completed'. (See below for explanation of statuses and an example of an expense request and report approval history).
+
+To check this by viewing the person expense approval history:
+
+      i. People/List/Type last name in Last Name field and click blue > then List
+      ii. Click on the dollar icon
+      iii. Click on 'Completed Expense Requests' to view
+
+2.  Review the REQUEST
+
+    - Click on the magnifying glass to view all details
+    - Check the project type and expense type are correct
+    - Approve or Disapprove as you see fit - you do NOT need to check ProDev usage - We do that as part of the project approval process before the request reaches you.
+
+3.  Review the REPORT
+
+    - Make sure there are no duplicates
+    - Click on the magnifying glass to view all details
+    - Check the amount, project type and expense type are correct
+    - Click on attachments to open the receipt and double check the details
+
+    Checklist
+
+          1. Receipt should be attached
+          2. Check project/expense type
+          3. Check the receipt date (should match date entered in Unanet)
+          4. Check the receipt amount (should match amount entered in Unanet)
+          5. Check the credit card number does not end in ****. If it does, then the receipt needs to be associated with our company credit card and not reimbursed. If it is the company credit card, click on 'disapprove' and ask the  requestor to resubmit with '**** Visa' for Payment Method (on the Details page)
```

Comment:
@farooqzakhilwal same indentation problem here I believe.

Reply:
should be good now?

Reply:
Na it is still showing up in a code snippet format. Maybe try the removing an additional 4 spaces of indentation?